aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,302 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization, Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto @bartmax <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website withe ASP.NET Core is not easy but it will allow your site to reach a wider audience. ASP.NET  has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
+
+This sample is based on the ``Localization.StarterWeb`` project on `GitHub <https://github.com/aspnet/entropy>`__.
+
+.. contents:: In this article:
+  :local:
+  :depth: 1
+  
+.. note:: Currently, resource files are not read when the project is run from Visual Studio. See `this issue <https://github.com/aspnet/dnx/issues/3047>`_ for more information.
+
+IStringLocalizer
+---------------------------------------------
+
+Introduced in ASP.NET Core, ``IStringLocalizer`` was architected to maximize productivity when developing localized apps. 
+
+The simple interface has an indexer and an ``IEnumerable`` for returning localized strings. Unlike previous localization approaches, ``IStringLocalizer`` doesn't require you to store the default language strings in a resource file. You can develop an app targeted for localization and not need to create resource files early in development. The recommendations is that you do **not** create a default language resource file. In previous version of ASP.NET, you had to move all your localization strings into a resource (.resx) file and change your app everywhere the strings were used, even for the default language. You no longer need to move your localized strings, you simply wrap them.
```

Comment:
Is the recommendation to not create a default language resource file? I think the guide should note that now this is **optional** and **not required**. But in some scenarios it is still recommended to use resource files.